modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -65,23 +65,27 @@ $ oc adm upgrade --to-multi-arch
 $ oc adm upgrade
 ----
 +
+.Example output
+
+[source,terminal]
+----
+working towards ${VERSION}: 106 of 841 done (12% complete), waiting on etcd, kube-apiserver
```

Comment:
Recorded from our [test](https://redhat-internal.slack.com/archives/CJ1J9C3V4/p1732148310689569?thread_ts=1729872364.520849&cid=CJ1J9C3V4) :

```suggestion
working towards ${VERSION}: 761 of 890 done (85% complete), waiting on machine-config
```

It is likely that that "machine-config" got caught in the output because MCO is the only cluster operator that implement the method described in the [enhancement](https://github.com/openshift/enhancements/pull/1690).